Firestore/src/BulkWriter.php

```diff
@@ -190,9 +234,72 @@ public function __construct(ConnectionInterface $connection, $valueMapper, $data
         $options += [
             'maxBatchSize' => self::MAX_BATCH_SIZE,
             'greedilySend' => true,
+            'isThrottlingEnabled' => true,
+            'initialOpsPerSecond' => null,
+            'maxOpsPerSecond' => null,
```

Comment:
What about defaulting these to the values defined by the constants above? It might help simplify some of the logic below.

Reply:
thanks for pointing that out